manuscript/4.0authentication.md

```diff
@@ -55,174 +55,174 @@ While generating a new session, we need to check if a sessions is already active
 Till now we never used any third party library or a framework in this book, this is for the first time that we are doing so, we better use libraries for handling sessions, since security is the primary aspect of any web application it is wiser to use existing (and good) packages to manage security.
 
 Path: `~/Tasks/sessions/sessions.go`
-
-    package sessions
-
-    import (
-        "net/http"
-
-        "github.com/gorilla/sessions"
-    )
-
-    //Store the cookie store which is going to store session data in the cookie
-    var Store = sessions.NewCookieStore([]byte("secret-password"))
-
-    //IsLoggedIn will check if the user has an active session and return True
-    func IsLoggedIn(r *http.Request) bool {
-        session, _ := Store.Get(r, "session")
-        if session.Values["loggedin"] == "true" {
-            return true
-        }
-        return false
-    }
-
+```golang
```

Comment:
I did see the markdown preview! Please look at the line numbers starting from 58, the markdown preview shows that the formatting is broken, it is just for one block :-)